<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DataDialog</class>
 <widget class="QDialog" name="DataDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>350</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Channel data</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="CustomTable" name="tableWidget">
     <row>
      <property name="text">
       <string>1</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>2</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>3</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>4</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>5</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>6</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>7</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>8</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>9</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>10</string>
      </property>
     </row>
     <column>
      <property name="text">
       <string>Width [m]</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Depth [m]</string>
      </property>
     </column>
     <item row="0" column="0">
      <property name="textAlignment">
       <set>AlignTrailing|AlignVCenter</set>
      </property>
     </item>
     <item row="0" column="1">
      <property name="textAlignment">
       <set>AlignTrailing|AlignVCenter</set>
      </property>
     </item>
     <item row="1" column="0">
      <property name="textAlignment">
       <set>AlignTrailing|AlignVCenter</set>
      </property>
     </item>
     <item row="1" column="1">
      <property name="textAlignment">
       <set>AlignTrailing|AlignVCenter</set>
      </property>
     </item>
     <item row="2" column="0">
      <property name="text">
       <string/>
      </property>
      <property name="textAlignment">
       <set>AlignTrailing|AlignVCenter</set>
      </property>
     </item>
     <item row="2" column="1">
      <property name="textAlignment">
       <set>AlignTrailing|AlignVCenter</set>
      </property>
     </item>
     <item row="3" column="0">
      <property name="textAlignment">
       <set>AlignTrailing|AlignVCenter</set>
      </property>
     </item>
     <item row="3" column="1">
      <property name="textAlignment">
       <set>AlignTrailing|AlignVCenter</set>
      </property>
     </item>
     <item row="4" column="0">
      <property name="textAlignment">
       <set>AlignTrailing|AlignVCenter</set>
      </property>
     </item>
     <item row="4" column="1">
      <property name="textAlignment">
       <set>AlignTrailing|AlignVCenter</set>
      </property>
     </item>
     <item row="5" column="0">
      <property name="textAlignment">
       <set>AlignTrailing|AlignVCenter</set>
      </property>
     </item>
     <item row="5" column="1">
      <property name="textAlignment">
       <set>AlignTrailing|AlignVCenter</set>
      </property>
     </item>
     <item row="6" column="0">
      <property name="textAlignment">
       <set>AlignTrailing|AlignVCenter</set>
      </property>
     </item>
     <item row="6" column="1">
      <property name="textAlignment">
       <set>AlignTrailing|AlignVCenter</set>
      </property>
     </item>
     <item row="7" column="0">
      <property name="textAlignment">
       <set>AlignTrailing|AlignVCenter</set>
      </property>
     </item>
     <item row="7" column="1">
      <property name="textAlignment">
       <set>AlignTrailing|AlignVCenter</set>
      </property>
     </item>
     <item row="8" column="0">
      <property name="textAlignment">
       <set>AlignTrailing|AlignVCenter</set>
      </property>
     </item>
     <item row="8" column="1">
      <property name="textAlignment">
       <set>AlignTrailing|AlignVCenter</set>
      </property>
     </item>
     <item row="9" column="0">
      <property name="textAlignment">
       <set>AlignTrailing|AlignVCenter</set>
      </property>
     </item>
     <item row="9" column="1">
      <property name="textAlignment">
       <set>AlignTrailing|AlignVCenter</set>
      </property>
     </item>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>File:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="openButton">
       <property name="text">
        <string>&amp;Open</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="saveButton">
       <property name="text">
        <string>&amp;Save</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Rows:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="addButton">
       <property name="text">
        <string>&amp;Add</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="removeButton">
       <property name="text">
        <string>&amp;Remove</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CustomTable</class>
   <extends>QTableWidget</extends>
   <header>customtable.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>tableWidget</tabstop>
  <tabstop>openButton</tabstop>
  <tabstop>saveButton</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DataDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DataDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
